### PR TITLE
filter small sv

### PR DESCRIPTION
**Pull request by kevinhkhuynh**
Filter small SV

### CODE REVIEW

**Comment by kevinhkhuynh** [comment]
What's a good value for this SV tolerance, or do we even want to use this? Setting SV to 1e-14 passes all tests, and doesn't filter anything as it's too small.

**Comment by chldkdtn** [comment]
> What's a good value for this SV tolerance, or do we even want to use this? Setting SV to 1e-14 passes all tests, and doesn't filter anything as it's too small.

@kevinhkhuynh Setting SV tolerance to 1e-14 for now is good to keep the current tests valid. Would you add one new test though that has a larger SV tolerate, such as 1e-6?

**Comment by kevinhkhuynh** [comment]
Ready for review.

**Review by chldkdtn — APPROVED** [review]
Looks good to me!

File: laghos_rom.hpp [hunk 1]
```diff
@@ -109,9 +114,10 @@ class ROM_Sampler
     {
         const int window = input.window;
 
-        // TODO: read the following parameters from input?
-        double model_linearity_tol = 1.e-7;
-        double model_sampling_tol = 1.e-7;
+        if (sampleF)
```
Comment (siuwuncheung) [inline laghos_rom.hpp]: This is no longer necessary. @dylan-copeland Is that right?

File: laghos.cpp [hunk 2]
```diff
@@ -173,6 +173,9 @@ int main(int argc, char *argv[])
     double sFactorX = 2.0;
     double sFactorV = 20.0;
     double sFactorE = 2.0;
+    double model_linearity_tol = 1.e-7;
```
Comment (dylan-copeland) [inline laghos.cpp]: These 3 variables are not used and should be removed.

File: laghos_rom.hpp [hunk 3]
```diff
@@ -78,6 +78,11 @@ struct ROM_Options
     int dimFv = -1;
     int dimFe = -1;
 
+    // Incremental SVD options
+    double model_linearity_tol = 1.e-7;
```
Comment (dylan-copeland) [inline laghos_rom.hpp]: I think these are the historical names of these variables. Maybe it is time to update them to something more descriptive. First, `model` is not helpful. Second, if they are only used for incremental SVD, how about something like `model_*` --> `incSVD_*`?

**Review by dylan-copeland — APPROVED** [review]

**Comment by chldkdtn** [comment]
@siuwuncheung Would you review this when you get a chance?

File: laghos_rom.hpp (hunk 1 above)
Comment (kevinhkhuynh) [reply laghos_rom.hpp]: removed

**Review by siuwuncheung — APPROVED** [review]

**Pull request merged**